tools/clean_demo_hosts.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -x
 
-for host in $(oc get baremetalhosts --no-headers | grep '^demo-' | awk '{print $1}')
+for host in $(oc get baremetalhosts --no-headers -l metalkubedemo | awk '{print $1}')
```

Comment:
I think you can skip the `for` loop and just do `oc delete baremetalhosts -l metalkubedemo`.

Reply:
Done